src/Microsoft.ML.Data/Transforms/TypeConverting.cs

```diff
@@ -156,7 +155,8 @@ private static VersionInfo GetVersionInfo()
                 modelSignature: "CONVERTF",
                 // verWrittenCur: 0x00010001, // Initial
                 // verWrittenCur: 0x00010002, // Added support for keyRange
-                verWrittenCur: 0x00010003, // Change to transformer leads to change of saving objects.
+                //verWrittenCur: 0x00010003, // Change to transformer leads to change of saving objects.
+                verWrittenCur: 0x00010004, // Removed Min and Contiguous from KeyCount.
```

Comment:
Do you have a test that loads a model with the old version to ensure backwards compat?

Reply:
I added one, and did the same for MatrixFactorization.